eng/pipelines/common/checkout-job.yml

```diff
@@ -22,3 +49,16 @@ jobs:
   - publish: $(Build.StagingDirectory)/Checkout.bundle
     artifact: Checkout_bundle
     displayName: Upload Checkout.bundle
+
+  - ${{ if and(ne(parameters.paths[0], ''), in(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
I thought that `Build.Reason` was a single variable, is it actually a list instead?

Reply:
It seems like you're right and it is a single variable. I thought it was a list because that's how arcade and at least the repos that I looked at use it, the use `in` and `notIn`. I will change it to `eq`.

Reply:
`in` is "first arg equal to any of the other args", not a list op. https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#in

So I would guess `in` was either intentional to make the conditions more similar, or snuck in when someone removed some options:

```
in(variables['Build.Reason'], 'PullRequest', 'BatchedCI', 'Schedule')
in(variables['Build.Reason'], 'PullRequest')
```

Reply:
You're right. That could've been it.

Reply:
Fixed.